<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GnuPGControllerDialog</class>
 <widget class="QDialog" name="GnuPGControllerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GnuPG Controller</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QCheckBox" name="asciiModeCheckBox">
               <property name="text">
                <string>Use Binary Mode for File Operations</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="gpgmeDebugLogCheckBox">
               <property name="text">
                <string>Enable GpgME Debug Log</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="killAllGnuPGDaemonCheckBox">
               <property name="text">
                <string>Kill all gnupg daemon at close</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Key Database</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTableWidget" name="keyDatabaseTable">
           <property name="editTriggers">
            <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragDropMode::NoDragDrop</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::DropAction::IgnoreAction</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Path</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addNewKeyDatabaseButton">
           <property name="text">
            <string>Add New Key Database</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Advanced</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QCheckBox" name="useCustomGnuPGInstallPathCheckBox">
             <property name="text">
              <string>Use Custom GnuPG</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="currentCustomGnuPGInstallPathLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="useCustomGnuPGInstallPathButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Select GnuPG Path</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="customGnuPGPathTipsLabel">
             <property name="text">
              <string>Tips: please select a directroy where &quot;gpgconf&quot; is located in.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="restartTipsLabel">
       <property name="text">
        <string>Tips: notice that modify any of these settings will cause an Application restart.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionRemove_Selected_Key_Database">
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionOpen_Key_Database">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open Key database by System File Expolrer</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionMove_Key_Database_To_Top">
   <property name="text">
    <string>Move To Top</string>
   </property>
   <property name="toolTip">
    <string>Move this Key Database Item to the top of the list</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionMove_Key_Database_Up">
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="toolTip">
    <string>Move this Key Database Item up</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionEdit_Key_Database">
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit this Key Database Item</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionMove_Key_Database_Down">
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="toolTip">
    <string>Move this Key Database Item down</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GnuPGControllerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GnuPGControllerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
